src/legacy/core_plugins/metrics/public/components/aggs/moving_average.js

```diff
@@ -135,29 +154,27 @@ const MovingAverageAggUi = props => {
       <EuiFlexGroup gutterSize="s">
         <EuiFlexItem>
           <EuiFormRow
-            id={htmlId('model')}
-            label={(<FormattedMessage
-              id="tsvb.movingAverage.modelLabel"
-              defaultMessage="Model"
-            />)}
+            id={htmlId('model_type')}
+            label={i18n.translate('tsvb.movingAverage.modelLabel', {
+              defaultMessage: 'Model',
+            })}
           >
             <EuiComboBox
               isClearable={false}
-              placeholder={intl.formatMessage({ id: 'tsvb.movingAverage.model.selectPlaceholder', defaultMessage: 'Select' })}
+              placeholder={i18n.translate('tsvb.movingAverage.model.selectPlaceholder', { defaultMessage: 'Select' })}
               options={modelOptions}
               selectedOptions={selectedModelOption ? [selectedModelOption] : []}
-              onChange={handleSelectChange('model')}
+              onChange={handleSelectChange('model_type')}
               singleSelection={{ asPlainText: true }}
             />
           </EuiFormRow>
         </EuiFlexItem>
         <EuiFlexItem>
           <EuiFormRow
             id={htmlId('windowSize')}
-            label={(<FormattedMessage
-              id="tsvb.movingAverage.windowSizeLabel"
-              defaultMessage="Window Size"
-            />)}
+            label={i18n.translate('tsvb.movingAverage.windowSizeLabel', {
+              defaultMessage: 'Window Size',
+            })}
```

Comment:
I think it's nice to have here a check on the `window` parameter in case of using the `holtWinters`.
From the docs:
> Unfortunately, due to the nature of Holt-Winters, it requires two periods of data to "bootstrap" the algorithm. This means that your window must always be at least twice the size of your period. 
It will be great to have a basic validation on those parameters before sending the request

Reply:
Nice catch. Help text is added